application/src/main/java/org/thingsboard/server/service/entitiy/user/DefaultUserService.java

```diff
@@ -82,8 +82,7 @@ public void delete(TenantId tenantId, CustomerId customerId, User tbUser, User u
         UserId userId = tbUser.getId();
 
         try {
-            tbAlarmService.unassignUserAlarms(tbUser.getTenantId(), tbUser, System.currentTimeMillis());
-            userService.deleteUser(tenantId, userId);
+            userService.deleteUser(tenantId, tbUser);
```

Comment:
I'm confused what the purpose **`user`** and **`tbUser`**.
I guess some user is a **victim** and another user is a **killer**
I understand it was before you. But please, add some clear naming for two user-related variables.
Will really appreciate you!